library/md2.c

```diff
@@ -243,6 +266,13 @@ int mbedtls_md2_ret( const unsigned char *input,
     return( ret );
 }
 
+void mbedtls_md2( const unsigned char *input,
```

Comment:
Blocker: These wrappers must be guarded by `! MBEDTLS_DEPRECATED_REMOVED`.

Reply:
I think `all.sh` should catch this - doesn't it include a run with `MBEDTLS_DEPRECATED_REMOVED` set?

Reply:
Fixed.

`all.sh` does not catch this.